exec/java-exec/src/test/java/org/apache/drill/TestBuilder.java

```diff
@@ -235,6 +240,20 @@ public CSVTestBuilder csvBaselineFile(String filePath) {
         expectedNumBatches);
   }
 
+  public SchemaTestBuilder schemaBaseLine(List<Pair<SchemaPath, TypeProtos.MajorType>> expectedSchema) {
+    assert expectedSchema != null : "The expected schema can be provided once";
+    assert baselineColumns == null : "The column information should be captured in expected schem, not baselineColumns";
```

Comment:
Could you add a few tests to the TestFrameworkTest that hit these conditions? It would be good if future changes to the test builder that may cause some currently-valid test configurations to fail (or currently invalid configurations to be allowed) would be detected by the tests for the framework itself.